csaf-import/src/main/kotlin/io/github/csaf/sbom/KoverIgnore.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024, The Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.github.csaf.sbom
+
+@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
+annotation class KoverIgnore(@Suppress("unused") val reason: String)
```

Comment:
maybe make this an internal visibility?

Reply:
Then we need to introduce it again for every library we want to use it in.
I was even thinking of the exact opposite: Moving this to some meta-package with common utils that we can include everywhere.

Reply:
Is it acceptable to keep it as-is for now?

Reply:
yes